ci-operator/step-registry/single-node/conf/e2e/single-node-conf-e2e-commands.sh

```diff
@@ -19,3 +19,33 @@ spec:
     systemReserved:
       memory: 3Gi
 EOF
+
+# Use first two cores on a single node for workload partitioning
+cat >"${SHARED_DIR}/manifest_single-node-workload-partitioning.yml" << EOF
```

Comment:
Does this enable workload partitioning on all e2e SNO tests or just the new ones being added?

Reply:
All SNO tests. AFAIK, the AWS SNO cluster only has two cores ( 4 virtual cores). Not sure about the bare metal SNO cluster.

Reply:
I think we would only want to enable workload partitioning for the tests we are adding. The number of cores needed by other tests for the platform will depend on how many/which operators are deployed. @browsell can you confirm?

Reply:
From https://github.com/openshift/enhancements/blob/master/enhancements/workload-partitioning/management-workload-partitioning.md#goals:
```
Clusters built in this way should pass the same Kubernetes and OpenShift conformance and functional end-to-end tests as single-node deployments that are not isolating the management workloads.
```
So the current SNO jobs need all CPUs to comply with the enhancement goals when the conformance tests are ran.